libs/atqr/domain/src/lib/domain-spec/aggregrates/Challenge Aggregate.disl

```diff
@@ -30,40 +31,41 @@ domain ATQR
           @deadline: Date
           @pledge: DISL.Money
             greater than 0
-          @deadline: FutureDate
```

Comment:
The right type for @deadline is FutureDate. I pass through that in another task and confirmed it with Saulo.